lib/zanzibar.rb

```diff
@@ -113,17 +113,26 @@ def get_secret(scrt_id, token = nil)
       raise "There was an error getting the secret with id #{scrt_id}: #{err}"
     end
 
-    ## Retrieve a simple password from a secret
+    ## Retrieve the value from a field label of a secret
     # Will raise an error if there are any issues
     # @param [Integer] the secret id
-    # @return [String] the password for the given secret
-
-    def get_password(scrt_id)
+    # @param [String] the field label to get
+    # @return [String] the value for the given field label
+    def get_fieldlabel_value(scrt_id, fieldlabel)
```

Comment:
If you give `fieldlabel` a default value of `'Password'`, you should be able to simplify this as well as the conditional in `get.rb`